usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -417,7 +434,50 @@ def query_elasticsearch_for_awards(self, time_periods: list) -> list:
         search = AwardSearch().filter(filter_query)
         self.apply_elasticsearch_aggregations(search)
         response = search.handle_execute()
-        overall_results = self.build_elasticsearch_result_awards(response.aggs, time_periods)
+        overall_results = self.build_elasticsearch_result_awards_subawards(response.aggs, time_periods)
+
+        return overall_results
+
+    def query_elasticsearch_for_subawards(self, time_periods: TimePeriodMetadata) -> list:
```

Comment:
If we are using the `TimePeriod` dataclass, then this type hint becomes a list of `TimePeriod`s
```suggestion
    def query_elasticsearch_for_subawards(self, time_periods: list[TimePeriod]) -> list:
```